cg-spec/editors_draft.html

```diff
@@ -508,40 +508,40 @@ <h2>Grammar</h2>
             <a  data-cite="TURTLE#grammar-production-collection">collection</a> `|`
             <a data-cite="TURTLE#grammar-production-blankNodePropertyList">blankNodePropertyList</a> `|`
             <a data-cite="TURTLE#grammar-production-literal">literal</a> `|`
-            <a style="font-weight: bold" href="#grammar-production-embTriple">embTriple</a>
+            <a style="font-weight: bold" href="#grammar-production-quotedTriple">quotedTriple</a>
           </td>
         </tr>
-        <tr style="font-weight: bold" id="grammar-production-embTriple">
+        <tr style="font-weight: bold" id="grammar-production-quotedTriple">
           <td>[27t]</td>
-          <td>`embTriple`</td>
+          <td>`quotedTriple`</td>
           <td>::=</td>
           <td>
             <code class="grammar-literal">'&lt;&lt;'</code>
-            <a href="#grammar-production-embSubject">embSubject</a>
+            <a href="#grammar-production-qTSubject">qTSubject</a>
```

Comment:
Minor style comment: I would have used `qtSubject`, not `qTSubject`. Ditto `qtObject`.

(In the SPARQL, there is `QTSubject, and generally, the case style is different so `QT` makes sense there.)

Reply:
Fixed in 217f5acbc14afd086b91598e47c57da2ea93f2d6.